src/v2/ion/uiSchemaTransformer.js

```diff
@@ -134,6 +134,19 @@ const createUISchema = (remediationValue = [], factors = [], authenticators = []
     const uiSchema = {
       type: 'text',
     };
+
+    if (ionFormField.name === 'rememberMe') { 
+      // For Remember Me checkbox, we need the label only on the right side of it.
+      uiSchema.placeholder = ionFormField.label;
+      uiSchema.label = false;
+      // Separating prop type for Backbone.Model
+      // from html input type
+      uiSchema.modelType = ionFormField.type;
+      // uiSchema type is the html input type desired.
+      uiSchema.type = 'checkbox';
+      uiSchema.required = false;
```

Comment:
when `required` is missing, default to `false`. would be better to ask to change API by adding `required: false`

Reply:
it doesn't appear to be false by default as error gets thrown.

Reply:
makes sense to do `uiSchema.required = ionFormField.required || false;` (it is not `false` by default right now)